Simplified/Book/UI/NYPLBookCellDelegate+Audiobooks.m

```diff
@@ -110,6 +111,11 @@ - (void)openAudiobook:(NYPLBook *)book withJSON:(NSDictionary *)json decryptor:(
 
       // poll audiobook player so that we can save the reading position
       [self scheduleTimerForAudiobook:book manager:manager viewController:audiobookVC];
+      
+      [[NSNotificationCenter defaultCenter] addObserver:self
+                                               selector:@selector(scheduleAudiobookProgressSavingTimer)
+                                                   name:UIApplicationWillResignActiveNotification
+                                                 object:nil];
```

Comment:
For some reason, calling the Swift function here and passing in the `manager` object ends up not calling the function at all. I use a bridge function on line 225 below in order to call the Swift function. Let me know if there is workaround/fix.